metrics/exact_match/README.md

```diff
@@ -0,0 +1,85 @@
+# Metric Card for Exact Match
+
+
+## Metric Description
+A given predicted string's exact match score is 1 if it is the exact same as its reference string, and is 0 otherwise.
+
+- **Example 1**: The exact match score of prediction "Happy Birthday!" is 0, given its reference is "Happy New Year!".
+- **Example 2**: The exact match score of prediction "The Colour of Magic (1983)" is 1, given its reference is also "The Colour of Magic (1983)".
+
+The exact match score of a set of predictions is the sum of all of the individual exact match scores in the set, divided by the total number of predictions in the set.
+
+- **Example**: The exact match score of the set {Example 1, Example 2} (above) is 0.5.
+
+
+## How to Use
+
+```python
+from datasets import load_metric
+exact_match_metric = load_metric("exact_match")
+results = exact_match_metric.compute(predictions=predictions, references=references)
+```
+
+### Inputs
+- **predictions**: Predicted labels, as returned by a model.
+- **references**: Ground truth labels.
+- **regexes_to_ignore**: List, defaults to None. Regex expressions of characters to ignore when calculating the exact matches. Note: the regex changes are applied after capitalization is normalized.
+- **ignore_capitalization**: Boolean, defaults to False. If true, turns everything to lowercase so that capitalization differences are ignored.
```

Comment:
I also agree that the metric cards should have an identical structure! So it seems we're all in agreement on that :)

The two sections that are in the squad card but that I left out here are 'Values from Popular Papers' and 'Further References'. I did this because they didn't seem as relevant in the case of exact match. If anyone has anything you think should be in those sections, feel free to add them in, or let me know what they are and I can add them in. If not, we can also keep them in for consistency across metric cards, but having empty sections seems kind of funky, so we should probably have some sort of note about why they're there but empty.

In terms of the Output Values/Examples/How to Use hierarchy, I did it this way because grouping Inputs, Output Values, and Examples under the How to Use heading seemed to make it more readable and therefore more accessible to those looking to understand the metric. I also think that having a detailed input section is important, because it makes it much easier to use the metric, especially for those that have less experience with our library or with coding in general. 

I also think it would be useful, once we fully decide on a format for the metric cards, to have a template file that can simply be duplicated and filled out whenever anyone needs to make a new metric card. I know we've talked about it in the past @sashavor, and that you've been against it, but I think it could prevent a lot of these issues of small differences between different cards.  

Let me know what you two think! :)

Reply:
I would keep an empty section in any case, even if there's nothing in it.

You can also look at the metrics that use `exact_match` under the hood (like SQuAD and maybe others?) and put the values in there, saying that this metric is actually used as part of the evaluation on ____ dataset.

Maybe we can have a README with the metric cards structure when we migrate to a separate metrics repo?

Reply:
> I would keep an empty section in any case, even if there's nothing in it.
>
> You can also look at the metrics that use exact_match under the hood (like SQuAD and maybe others?) and put the values in there, saying that this metric is actually used as part of the evaluation on ____ dataset.

+1 on this ^^

> Maybe we can have a README with the metric cards structure when we migrate to a separate metrics repo?

Sure, or feel free to add one in the `/templates` directory for now :)